cmd/ncproxy/ncproxy.go

```diff
@@ -463,6 +463,17 @@ func (s *grpcService) AddEndpoint(ctx context.Context, req *ncproxygrpc.AddEndpo
 			}
 			return nil, errors.Wrapf(err, "failed to get endpoint with name `%s`", req.Name)
 		}
+		if req.AttachToHost {
```

Comment:
Should we instead return an error if `AttachToHost` and `req.NamespaceID` are both set? So that it's more clear to the caller that the namespace ID field is not used in that case.

Reply:
returning an error seems extreme, but ill output a warning log

Reply:
This log will only really be on our side though, so the caller may not know. I don't think it's blocking as is though.

Reply:
ohh yeah, true
we can probably make it fail in that case in a future PR